src/util/hass-util.html

```diff
@@ -69,6 +71,16 @@
   }
 };
 
+// If root has lastChild, set 'isVisible' attribute of that child to false.
+window.hassUtil.resetDynamicContent = function (root) {
```

Comment:
I think that you should inline this logic into `more-info-content.html`, it's the only place that knows about `isVisible`. Alternative, we could decide to just remove the whole `more-info` tag from the DOM when there is no stateObj, so there won't even be a need for `isVisible`